spikes/lk_config_kit/main.cpp

```diff
@@ -0,0 +1,72 @@
+// Leka - LekaOS
+// Copyright 2020 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+#include <filesystem>
+
+#include "drivers/BufferedSerial.h"
+#include "rtos/ThisThread.h"
+
+#include "ConfigKit.h"
+#include "FATFileSystem.h"
+#include "FileSystemKit.h"
+#include "HelloWorld.h"
+#include "LogKit.h"
+#include "SDBlockDevice.h"
+
+using namespace leka;
+using namespace std::chrono_literals;
+
+SDBlockDevice sd_blockdevice(SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK);
+FATFileSystem fatfs("fs");
+
+auto _configuration_file									   = FileSystemKit::File {};
+const std::filesystem::path bootloader_battery_hysteresis_path = "/fs/conf/bootloader_battery_hysteresis.conf";
+auto bootloader_battery_level_hysteresis					   = Config(bootloader_battery_hysteresis_path);
+auto _config_manager										   = ConfigKit();
```

Comment:
ici pourquoi tu utilises pas ta nouvelle config toute faite?

`_configuration_file` ne semble servir à rien



```suggestion
auto configkit = ConfigKit();
```